### DISCUSSION TITLE
Uninstall in Windows

### DISCUSSION THREAD

**Discussion by rsmcgowan** [MiKTeX/miktex#1359] [General]
How do I cleanly uninstall MiKTeX, including the console in Windows. My Console comes up with an internal error when I attempt to uninstall.

**Comment by timetraveller2014** [comment]
press WIndows+R, type appwiz.cpl, press enter, now uninstall MikTex!

**Comment by timetraveller2014** [comment]
> How do I cleanly uninstall MiKTeX, including the console in Windows. My Console comes up with an internal error when I attempt to uninstall.

i was trying it since last week, i just got it and i came here!
